GapsWeb/src/main/java/com/jasonhhouse/gaps/service/GapsSearchService.java

```diff
@@ -885,6 +921,33 @@ private void handleCollection(Movie movie, Gaps gaps, OkHttpClient client, JSONO
         searched.add(movie);
     }
 
+    private void writeRssFile(List<Movie> recommended) {
+        JSONArray jsonRecommended = new JSONArray();
+
+        try {
+            File file = new File("rssFeed.json");
+            file.createNewFile();
+            FileWriter writer = new FileWriter(file);
+
+            for (Movie mov : recommended) {
+
+                JSONObject obj = new JSONObject();
+                obj.put("imdb_id", mov.getImdbId());
+                obj.put("tvdb_id", mov.getTvdbId());
+                obj.put("title", mov.getName());
+                obj.put("release_date", mov.getYear());
+                obj.put("poster_path", mov.getPosterUrl());
+                jsonRecommended.put(obj);
+            }
+
+            jsonRecommended.write(writer);
+            writer.flush();
+            writer.close();
+        } catch (Exception e) {
```

Comment:
The writer isn't closed if it throws an exception. Can't you use the java try closeable here?